forch/forchestrator.py

```diff
@@ -34,7 +34,7 @@
 _FAUCET_CONFIG_DEFAULT = 'faucet.yaml'
 _DEFAULT_PORT = 9019
 _PROMETHEUS_HOST = '127.0.0.1'
-_LOG_FORMAT = '%(asctime)s %(name)-8s %(levelname)-8s %(message)s'
+_LOG_FORMAT = '%(asctime)s %(name)-8s %(levelname)-8s %(filename)s:%(lineno)s - %(funcName)20s() %(message)s'
```

Comment:
not convinced this is a good idea -- kinda spammy log messages.  Maybe remove for now?

Reply:
I agree. That was just a temporary change I made to make debugging easier.

Reply:
Done